pom.xml

```diff
@@ -781,31 +774,25 @@
     <profile>
       <!-- This profile is used to enable GraalVM native image testing -->
       <id>native</id>
-
       <dependencies>
-
         <dependency>
-          <groupId>org.junit.vintage</groupId>
-          <artifactId>junit-vintage-engine</artifactId>
-          <version>5.9.2</version>
-          <scope>test</scope>
-        </dependency>
-
-        <dependency>
-          <groupId>org.graalvm.buildtools</groupId>
-          <artifactId>junit-platform-native</artifactId>
-          <version>0.9.20</version>
-          <scope>test</scope>
+          <groupId>org.opentest4j</groupId>
```

Comment:
Could we add a source comment for this dependency addition?

Reply:
done.